### DISCUSSION TITLE
Very serious question: How is Mu pronounced?

### DISCUSSION THREAD

**Discussion by ajsirota** [mu-editor/mu#1447] [Q&A]
Is it "moo", like the sound a cow makes, or "myoo" like a cat "mews"?

**Comment by ZanderBrown** [comment]
Simplist answer: [Here is @ntoll saying Mu](https://youtu.be/bBNSkhIqG7s)

**Reply by dhalbert** [reply]
so mu (myoo), the Greek letter.

**Reply by ntoll** [reply]
Yes... so... erm... this is intentionally ambiguous. As @dhalbert notes, I say it like a mathematician might say the Greek letter μ... In modern Greek it's actually pronounced like the English word "me". So, the website (codewith.mu) could be pronounced as "code with me". Apparently in some dialects of ancient Greek it was pronounced as "moo" (hence the :cow: emoji use on social media). Being a former teacher and a little mischievous, I rather like the thought of my former teaching colleagues having to say "Mu" in front of a class of 11yo (who will all, inevitably, respond with farm yard noises). I firmly believe that school can be TOO serious, and silliness can help learning.

I also named it "mu" (無) as in the traditional retort to a famous Zen koan (Joshu's Mu - which starts the "Gateless Gate" collection of Koans - https://www.heartofmeditation.com/mumon-case1.html). That sort of Mu is a concept of nothingness... neither yes nor no. Doug Hofsdater, in "Godel, Escher, Bach" (where I first encountered the koan) describes it as an answer that un-asks the question to which it responds. I like how computing is mostly about logical ons and off, and Mu is neither. ;-) Also, the Chinese pronounce that symbol as "wu" and, apparently, it also can mean dancing (I like the thought of folks dancing with code in Mu).

Finally, I was originally going to call the editor "micro:ed", but someone at the BBC got shirty about it ("that's far too close to micro:bit", "but it's an editor FOR the micro:bit" etc)... and so I looked up what the symbol for "micro" units of measurement were... and it is, of course, the Greek letter mu, and so this mega-Yak shaving of name-related fun commenced. :-P

Anyway... you did ask... ;-) I hope this answers both your pronunciation question AND sheds some light on why I called in Mu.

To be honest, I rather hope you just say it however you like so long as it's done with kindness and humour..! :+1: For example, it's traditional in the Python community to pronounce "PyPI" (The Python Package Index) as "Cheeseshop" (don't ask... it's a long story). So you may pronounce "Mu" as "Collywobble" and still be correct... it's all in the intent of the saying. ;-)
